apps/chat/src/components/Chat/ConversationSettingsModels.tsx

```diff
@@ -79,30 +90,16 @@ export const ConversationSettingsModel = ({
               </div>
             </button>
           )}
-          {mappedEntities.map((entity) => (
-            <button
-              className={classNames(
-                'flex items-center gap-3 rounded border p-3 text-left text-xs',
-                modelId === entity.id && !replay.replayAsIs
-                  ? 'border-accent-primary'
-                  : 'border-primary hover:border-hover',
-              )}
-              key={entity.id}
-              onClick={() => onModelSelect(entity.id)}
-            >
-              <ModelIcon entityId={entity.id} entity={entity} size={24} />
-              <div className="flex flex-col gap-1">
-                <span data-qa="entity-name">{entity.name}</span>
-                {entity.description && (
-                  <span className="text-secondary" data-qa="entity-descr">
-                    <EntityMarkdownDescription isShortDescription>
-                      {entity.description}
-                    </EntityMarkdownDescription>
-                  </span>
-                )}
-              </div>
-            </button>
-          ))}
+          <ModelList
+            entities={enitities}
+            heading={''}
```

Comment:
maybe undefined better in such case than empty heading? According to normal logic you should render heading rather if it empty string, but not need to render if it's undefined